front/src/utils/timeManipulation.ts

```diff
@@ -16,6 +16,10 @@ export function ms2sec(ms: number) {
   return ms / 1000;
 }
 
+export function ms2min(ms: number) {
```

Comment:
nit: I prefer msToMin
```suggestion
export function msToMin(ms: number) {
```

Reply:
Note, we have some duplication between `physics.ts` (uses `XToY` style) and `timeManipulation.ts` (uses `X2Y` style). Would be nice to unify these, but it's orthogonal to this PR.

Reply:
Fine, we can refacto this in a future PR

Reply:
Okay noted on my side